### PR TITLE
:broom: Accommodate to knative-sandbox/reconciler-test#294

### PR DESCRIPTION
**Pull request by cardil**
## Proposed Changes

- :broom: Accommodate to knative-sandbox/reconciler-test#294

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -36,3 +36,10 @@ require (
 	knative.dev/reconciler-test v0.0.0-20220705155206-f05db88effbe
 	sigs.k8s.io/yaml v1.3.0
 )
+
+replace (
+	// FIXME: remove this pin when knative/eventing#6451 is merged
+	knative.dev/eventing => github.com/cardil/knative-eventing v0.11.1-0.20220718191420-28d8469b82bf
+	// FIXME: remove this pin when knative-sandbox/reconciler-test#294 is merged
+	knative.dev/reconciler-test => github.com/cardil/knative-reconciler-test v0.0.0-20220715181934-169264f8ba23
```
[the remark this answers is not in the document]
Comment (cardil) [reply go.mod]: Fixed with 79b576cde4b1ddfce028d049ef4d6934bbb38ef5

/unhold

**Comment by pierDipi** [comment]
@caniszczyk can you rebase please?

**Comment by pierDipi** [comment]
> caniszczyk can you rebase please?

sorry, wrong GH username :(

**Comment by cardil** [comment]
* [build #1559480858795053056](https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative-sandbox_eventing-kafka-broker/2413/integration-tests-namespaced-broker_eventing-kafka-broker_main/1559480858795053056) is likely a flake
```
/test integration-tests-namespaced-broker_eventing-kafka-broker_main
```
* [build #1559480859176734720](https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative-sandbox_eventing-kafka-broker/2413/channel-integration-tests-sasl-plain_eventing-kafka-broker_main/1559480859176734720) looks to me like an infra problem
```
/test channel-integration-tests-sasl-plain_eventing-kafka-broker_main
```

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Comment by cardil** [comment]
Again, this repo seams to have a lot of flakes

/retest

**Pull request merged**